spec/support/models/agency.rb

```diff
@@ -4,5 +4,6 @@
 class Agency
   include Mongoid::Document
   include Mongoid::Timestamps::Updated
+  include Mongoid::Attributes::Dynamic
```

Comment:
Instead of doing this which is rather invasive can you please either 1) find an existing model that already includes Dynamic and, if needed, add a datetime field there or 2) create a new model that contains the fields you need for these tests.

Reply:
Done > have changed to Book without any model modifications